locales/ru.yml

```
@@ -0,0 +1,200 @@
+no_favorites: "Пока нет избранного."
+no_tracks_found: "Треки не найдены."
+unknown_artist: "Неизвестный исполнитель"
+unknown_album: "Неизвестный альбом"
+unknown_title: "Неизвестное название"
+unknown_genre: "Неизвестно"
+most_played_local_tracks: "Ваши самые прослушиваемые локальные треки."
+no_tracks_in_library: "В вашей библиотеке не найдено треков."
+no_songs_here: "Здесь нет песен."
+syncing_with_server: "Синхронизация с сервером..."
+most_played_tracks: "Ваши самые прослушиваемые треки."
+no_more_songs: "Больше нет песен в очереди"
```

YAMLlint (1.38.0)

[error] duplication of key "no_more_songs" in mapping

(key-duplicates)